File: 7-kyu/the-office-i---outed/python/solution.py
def outed(meet, boss):
    total = 0
    for k, v in meet.items():
        if k == boss:
            total += 2 * v
        else:
            total += v
    result = total / len(meet.items())
    return 'Get Out Now!' if result <= 5 else 'Nice Work Champ!'

File: 7-kyu/the-office-i---outed/javascript/solution.js
const outed = (meet, boss) => {
  let sum = 0;
  for (const [employee, happiness] of Object.entries(meet)) {
    if (employee === boss) {
      sum += 2 * happiness;
    } else {
      sum += happiness;
    }
  }
  const mean = sum / Object.keys(meet).length;
  return mean <= 5 ? "Get Out Now!" : "Nice Work Champ!";
};
